modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/components/OrderableTable.tsx

```diff
@@ -242,9 +258,13 @@ const OrderableTable = ({
 		setItems(
 			query
 				? initialItems.filter((item) =>
-						fields.some((field) =>
-							String(item[field.name]).match(regexp)
-						)
+						fields.some((field) => {
+							if (field.contentRenderer) {
+								return false;
+							}
```

Comment:
With a content renderer, one can't know for sure what's the right value to query against. So, as an initial approach, this short-circuit logic looks fine to me. Nevertheless, users may find it odd that typing "Ascending" or "Descending" in the search box returns nothing. This needs some extra thought (for sure in a followup task) to come up with a way for the content renderers to make them self-findable.

Reply:
This is also an issue in base FDS. Search is run against entries in backend, and only original field values are checked. I think we should implement the same behavior here. Instead of adding the check for `field.contentRenderer`, I'd leave code as it is, matching original value. The reasoning I'd make is that this will cover some cases of custom renderers. Usually you do want to display original value, but in some special way. It's not ideal, but it's better than nothing.

We can try to improve that, match against real content of renderer, but that sounds pretty tricky to implement. As Dani said, we can try it in a followup task.

Reply:
I'll remove my change.

I thought it could have been strange seeing matches without certain text being highlighted, but I agree it could be helpful than not seeing any results.

My initial idea was having a separate property like "textToMatch" that would return a string for matching to the query, but I was hesitant in adding yet another property. We can think more in a followup task.